Makefile.toml

```diff
@@ -311,6 +322,11 @@ dependencies = [
     "gen-dart-feature",
     "gen-dart-example",
 ]
+[tasks.gen-kotlin]
```

Comment:
nitL these tasks should be made a part of the corresponding gen/test tasks

(gen, gen-feature, test, test-feature)

Reply:
I added it, but while on the issue of generation. I made diplomat generate the whole kotlin project, so when I run the duckscript it deletes the whole kotlin feature project directory including the tests. Consequently I have to check out the test files before running the tests. Looking now at the other languages it seems like the project files are out of scope. I have to admit I prefer the idea of a tool that will spit out a ready library that's all packaged, but understand if that's not how y'all want to do it. If I can keep it generating the whole project, then I'd like to modify the duckscript so that it copies over a configured subdirectory (that defaults to the target directory for generation)  from the generated files in `tmp`.

Reply:
That would be fine if you document it well enough.

Reply:
Great! I added a new argument to the duckscript helper at the end with several explanatory comments. I also mention that the kotlin gen task only copies over the main source files and not the generated project files for now even though the backend creates them.